### PR TITLE
Attempt to fix set-up, pin docker base image, and improve hokusai dev mode

### PR DESCRIPTION
**Pull request by joeyAghion**
For your consideration: hopeful fixes for both development environment and release-ability.

The https://github.com/artsy/aprd/pull/134 release encountered errors in production, which a revert (https://github.com/artsy/aprd/pull/138) didn't resolve. We restored production (and staging) by redeploying the previously built `production--2020-08-21--16-06-21` image. However, `master` is still undeployable. The error matches [this stackoverflow thread](https://stackoverflow.com/questions/62162329/httpoison-request-5-raises-httpoison-error-with-weird-reason):

> sslv3 was removed from Erlang 23.0 because it is an old and insecure protocol. You need to upgrade hackney to 1.16.0 or higher so that you can get this PR that fixes that issue:
> 
> https://github.com/benoitc/hackney/pull/619

This PR pins the docker base image at `alpine-elixir-phoenix:1.10.3` instead of `...latest` (which gets frequent Erlang and Elixir upgrades). I don't know the best practice about versioning in Phoenix apps, but it seems failure-prone to use `latest` because of errors like this.

I couldn't repro the error, presumably because local development doesn't use SSL. I went ahead and upgraded the `hackney` dependency as described in that thread anyway:

```
mix deps.update hackney
```

I was able to start a server (ignoring rabbitMQ stuff) and run tests both locally and in docker after these changes, but of course I'll want to verify carefully in staging.

Tangents:
* Local docker-based development wasn't usable so I tried to update `development.yml` based on `test.yml`, but I never got the database correctly hooked up and suspect there's a command I'm missing.
* I had to fiddle with local Erlang and Elixir versions a lot via `asdf` locally (described [here](https://artsy.slack.com/archives/C9YNS4X32/p1602529224443700)). Some of this should be incorporated into the README or automated, but I'm not sure the most conventional way to declare those in Elixir projects. I did commit a `.tool-versions` file with versions matching the `Dockerfile`'s image since that's unobtrusive. Let me know if you have suggestions.
* Related: the README implies a `.env` file is respected, but I found that a wrapper (like `foreman` or `dotenv`) was necessary. Am I missing something, or should I just clarify the docs?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM bitwalker/alpine-elixir-phoenix:latest
+FROM bitwalker/alpine-elixir-phoenix:1.10.3
```
Comment (ashkan18) [inline Dockerfile:1]: my bad 🤦

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ APRd (aka. APR dashboard), is a real-time dashboard built in [Elixir](https://el
 - Staging: https://aprd-staging.artsy.net
 - GitHub: https://github.com/artsy/aprd/
 - CI: [CircleCI](https://circleci.com/gh/artsy/apr-dashboard); merged PRs to artsy/apr-dashboard#master are automatically deployed to staging. PRs from `staging` to `release` are automatically deployed to production. [Start a deploy...](https://github.com/artsy/apr-dashboard/compare/release...staging?expand=1)
-- Point People: [@ashkan18][ashkan18], [@zephraph][zephraph]
+- Point People: [@jpotts244](https://github.com/jpotts244), [@zephraph][zephraph]
```
Comment (ashkan18) [inline README.md:14]: 💜

File: hokusai/development.yml [hunk 3]
```diff
@@ -5,5 +5,26 @@ services:
...
       file: build.yml
       service: aprd
+    env_file: ../.env
+    environment:
+    - DB_HOST=aprd-db
+    - RABBITMQ_HOST=aprd-rabbitmq
+    - RABBITMQ_USER=guest
+    - RABBITMQ_PASSWORD=guest
+    - RABBITMQ_PORT=5672
+    - SECRET_SALT=https://gph.is/2jAtlPQ
```
Comment (ashkan18) [inline hokusai/development.yml:15]: 🧂

**Review by ashkan18 — COMMENTED** [review]
I was here to copy a config to a new project 💜

**Comment by ashkan18** [comment]
> I had to fiddle with local Erlang and Elixir versions a lot via asdf locally (described here). Some of this should be incorporated into the README or automated, but I'm not sure the most conventional way to declare those in Elixir projects. I did commit a .tool-versions file with versions matching the Dockerfile's image since that's unobtrusive. Let me know if you have suggestions.

You can set the Elixir version in [`mix.exs`](https://github.com/artsy/aprd/blob/master/mix.exs#L8)

File: README.md (hunk 2 above)
Comment (jpotts244) [reply README.md:14]: ^^ big shoes to fill here!

**Review by jpotts244 — APPROVED** [review]

**Review by bhoggard — APPROVED** [review]

**Comment by jpotts244** [comment]
Wanted to take a peek at staging before merging this PR to keep an 👁️  on differences in behavior.

Seeing some looping behavior going on on the dashboard page [staging]( https://aprd-staging.artsy.net/)
which are corresponding to this sentry exceptions
https://sentry.io/organizations/artsynet/issues/1532516889/?project=1776903&query=is%3Aunresolved

Looks like it has to do with unhandled `nil`s, assuming this has to do with the amount of data we have access to on staging, looks like it has been happening for quite a while. Perhaps something we can clean up in a separate stream of work. 


Will merge this!

**Pull request merged**